package.json

```diff
@@ -90,15 +90,14 @@
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "chalk": "^2.4.1",
-    "color": "^3.0.0",
```

Comment:
This doesn't appear to be depended on anywhere (apart from `bpk-tokens` which has it's on dep).